nebula-exchange/src/test/resources/application.conf

```diff
@@ -224,23 +224,23 @@
     }
 
     # KAFKA
-    {
-      name: tag7
-      type: {
-        source: kafka
-        sink: client
-      }
-      service: "kafka.service.address"
-      topic: "topic-name"
-      fields: [kafka-field-0, kafka-field-1, kafka-field-2]
-      nebula.fields: [nebula-field-0, nebula-field-1, nebula-field-2]
-      vertex: {
-        field: kafka-field-0
-      }
-      partition: 10
-      batch: 10
-      interval.seconds: 10
-    }
+    # {
```

Comment:
please roll back these changes

Reply:
A new check is added in the config parse that an exception would be throw if any other config define after kafka, see Config.scala. However, there is two kafka defined in  the application.conf. If I don't comment this section, the test would not  pass.

Reply:
Ok， I'll split the config of Kafka out and use a single config file for Kafka later.